vmas/simulator/sensors.py

```diff
@@ -125,7 +132,7 @@ def render(self, env_index: int = 0) -> "List[Geom]":
                 ray.add_attr(xform)
 
                 ray_circ = rendering.make_circle(0.01)
-                ray_circ.set_color(*self._render_color.value)
+                ray_circ.set_color(*self.render_color)
```

Comment:
This just affects the ray circ and not the line, your problem seems to be the line